c/tests/pn_test.hpp

```diff
@@ -129,7 +130,7 @@ struct handler {
 struct driver : public ::pn_connection_driver_t {
   struct handler &handler;
 
-  driver(struct handler &h);
+  explicit driver(struct handler &h);
```

Comment:
I got it from clang-tidy and https://google.github.io/styleguide/cppguide.html#Implicit_Conversions. The widely accepted rule seems to put `explicit` everywhere, unless there is a very good reason not to. (And I can do this freely here, because I am not touching public API.)

Reply:
Ah, I'd forgotten the new iinitialisation syntax thing - that makes this much more compelling as a rule. It's definitely the better default anyway - but also makes conversion operators safer (even if it would make for widespread changes!).
Making this change to the exported include files would change the API, so we can't do it easily there (but actually not the ABI I think)